api/service/syncing/syncing.go

```diff
@@ -334,8 +334,12 @@ func (ss *StateSync) GetConsensusHashes(startHash []byte, size uint32) bool {
 			wg.Add(1)
 			go func() {
 				defer wg.Done()
-				response := peerConfig.client.GetBlockHashes(startHash, size)
+				response := peerConfig.client.GetBlockHashes(startHash, size, ss.selfip, ss.selfport)
```

Comment:
selfip, and selfport, why?

Reply:
This is to add ip and port (port is used for local debug) in the request, so that the server side know who is requesting. It is used for debugging purpose.